### PR TITLE
Copy updates

### PR DESCRIPTION
**Pull request by localastronaut**
Cleaned up and updated descriptions. Removed redundant “about” copy
from Auth homepage and gave it a fresh image.

### CODE REVIEW

**Comment by localastronaut** [comment]
Do you know what is causing the avatar bug? Looks like it has something to do with users who either imported their image from facebook or uploaded it themselves.   
Screen Shot 2019-10-12 at 2 33 38 PM

File: templates/home.html [hunk 1]
```diff
@@ -150,17 +91,17 @@ <h5><a href="{% url 'resources' %}">Learn More</a></h5>
                 <hr>
             </div>
             <div id="column-wrapper">
-                <h3 class="title">Free for artists</h3>
-                <h5 class="">Distribution, Funding, Payment Splits, and everything else you'll ever need</h5>
+                <h3 class="title">Start Investing</h3>
+                <h5>Buy shares in music. Create your muisc portfolio. Earn Money</h5>
```
Comment (RevolutionTech) [inline templates/home.html]: Typo
```suggestion
                <h5>Buy shares in music. Create your music portfolio. Earn Money</h5>
```

File: templates/home.html [hunk 2]
```diff
@@ -150,17 +91,17 @@ <h5><a href="{% url 'resources' %}">Learn More</a></h5>
...
                 <div class="center extra-padding">
-                    <a class="button sign-up uppercase" href="{% url 'artist_application' %}">Apply Now</a>
+                    <a class="button sign-up uppercase" href="{% url 'register' %}?next={{ request.get_full_path }}">Sign Up</a>
                 </div>
             </div>
             <div>
-                <h3 class="title">$1 per trade</h3>
-                <h5>Buy shares in music projects with just one click</h5>
+                <h3 class="title">For Artists</h3>
+                <h5 class="">Funding, Distribution, Payments, and everything else you'll ever need</h5>
```
Comment (RevolutionTech) [inline templates/home.html]: nit: Remove empty class attribute:
```suggestion
                <h5>Funding, Distribution, Payments, and everything else you'll ever need</h5>
```

**Comment by localastronaut** [comment]
Awesome. I committed those typo suggestions. Let me know if they went through

**Pull request merged**